diff --git a/home/friz64/xorg-xwayland-main/PKGBUILD b/xorg-xwayland-libdecor/PKGBUILD
index 1464cd9..c569e31 100644
--- a/home/friz64/xorg-xwayland-main/PKGBUILD
+++ b/xorg-xwayland-libdecor/PKGBUILD
@@ -1,6 +1,6 @@
 # Maintainer: AndyRTR <andyrtr@archlinux.org>
 
-pkgname=xorg-xwayland
+pkgname=xorg-xwayland-libdecor
 pkgver=23.2.6
 pkgrel=1
 arch=('x86_64')
@@ -34,9 +34,6 @@ makedepends=('meson' 'xorgproto' 'xtrans' 'libxkbfile' 'dbus'
 source=(https://xorg.freedesktop.org/archive/individual/xserver/xwayland-$pkgver.tar.xz{,.sig})
 sha512sums=('9e3c2253af335a559d0f890fa8f9bc381beca6531e0842d739ac15cbca008b3d07c0eefafd03611b04917c626861a7871a83657afa2a298994f4b162f714fc49'
             'SKIP')
-provides=('xorg-server-xwayland')
-conflicts=('xorg-server-xwayland')
-replaces=('xorg-server-xwayland')
 #validpgpkeys=('B09FAF35BE914521980951145A81AF8E6ADBB200') # "Michel Daenzer <michel@daenzer.net>"
 validpgpkeys=('67DC86F2623FC5FD4BB5225D14706DBE1E4B4540') # "Olivier Fourdan <fourdan@xfce.org>"
 validpgpkeys+=('3C2C43D9447D5938EF4551EBE23B7E70B467F0BF') # Peter Hutterer (Who-T) <office@who-t.net>
@@ -52,7 +49,8 @@ build() {
     -D xwayland_eglstream=true \
     -D glamor=true \
     -D xkb_dir=/usr/share/X11/xkb \
-    -D xkb_output_dir=/var/lib/xkb
+    -D xkb_output_dir=/var/lib/xkb \
+    -D libdecor=true
 
   # Print config
   meson configure build
@@ -61,11 +59,7 @@ build() {
 
 package() {
 
-  DESTDIR="${pkgdir}" ninja -C build install
-  # xorg-server-common file /usr/lib/xorg/protocol.txt
-  rm "${pkgdir}"/usr/lib/xorg/protocol.txt
-  rmdir "${pkgdir}"/usr/lib/xorg
-  rm "${pkgdir}"/usr/share/man/man1/Xserver.1
+  install -m755 -D build/hw/xwayland/Xwayland "${pkgdir}/usr/bin/Xwayland-libdecor"
 
   # license
   install -m644 -Dt "${pkgdir}/usr/share/licenses/${pkgname}" xwayland-$pkgver/COPYING
